cmd/payload-testing-ui/server.go

```diff
@@ -125,8 +160,36 @@ type server struct {
 	runsListTemplate *template.Template
 }
 
+func prLink(pr *prpqv1.PullRequestUnderTest) template.HTML {
+	org := template.HTMLEscapeString(pr.Org)
+	repo := template.HTMLEscapeString(pr.Repo)
+	title := template.HTMLEscapeString(pr.PullRequest.Title)
+	n := pr.PullRequest.Number
+	ret := fmt.Sprintf(`#<a href="http://github.com/%s/%s/pull/%d" title="%s" class="info">%d</a>`,
+		org, repo, n, title, n)
```

Comment:
The specific pull request will likely be more relevant here than in the Prow job list, so let's include the full title in text instead of in the `title` attribute, making sure it does not cause the `ci/…` items in the "name" column to be broken into more than one line.